var/spack/repos/builtin/packages/poplddecay/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Poplddecay(MakefilePackage):
+    """
+    PopLDdecay: a fast and effective tool for linkage disequilibrium
+    decay analysis based on variant call format files
+    """
+
+    homepage = "https://github.com/BGI-shenzhen/PopLDdecay"
+    url      = "https://github.com/BGI-shenzhen/PopLDdecay/archive/v3.41.tar.gz"
+
+    maintainers = ['robqiao']
+
+    version('3.41', sha256='09a1ad01581520b84ef73eaa0b199879c84e77b259ba6ff06dbca5fcfc090457')
+    version('3.40', sha256='5070930166fb90f7eaaa4b87c4430caa8a827d79c54683e2f56434a4daf69778')
+
+    def build(self, spec, prefix):
+        ###################
+        # Build Directory #
+        ###################
+        with working_dir('src'):
+            make()
+            make('clean')
```

Comment:
I forget if I asked this already, but is `make clean` actually required for the installation to succeed? If not this whole method can be replaced by:
```python
build_directory = 'src'
```

Reply:
@adamjstewart It cleans a bunch of *.o, so that simplify install steps, otherwise explict install steps are needed as in this update. Either way, I think they are eq in term of efforts.